extract_endpoint/tests/test_crypt_utils.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from crypt_utils import sign_string
+
+
+@pytest.fixture
+def sample_key() -> str:
+    return 'sample key'
+
+
+@pytest.fixture
+def sample_salt() -> str:
+    return 'sample salt'
+
+
+@pytest.fixture
+def sample_data() -> str:
+    return 'sample data'
+
+
+def test_sign_string(sample_salt: str, sample_key: str, sample_data: str) -> None:
+    actual = sign_string(salt=sample_salt, key=sample_key, data=sample_data)
+    assert actual == '8a2dbe236ead75dd242d3ee015b3d8b6'
```

Comment:
Nitpick, but this is a rather fragile test. Not sure what the best approach would be (assert it exists and is a string of a certain length?). Not a big deal.

Reply:
Yeah, that's not a bad idea. Also added tests ensuring that changing either the salts or the keys give different signatures.